### ISSUE TITLE
Voiceover does not announce the link information while navigating the document in Landscape mode.

### ISSUE THREAD

**Issue by msftedad** [gdh1995/pdf.js#18]
Attach (recommended) or Link to PDF file here:
https://mozilla.github.io/pdf.js/web/viewer.html.
[edad (2).pdf](https://github.com/gdh1995/pdf.js/files/14599774/edad.2.pdf)

Configuration:
- Operating system and its version: iPhone 13 Version-17.4
- PDF.js version: 1.4

Pre-requisites:
Access the Control Center by swiping downward from the upper-right corner of the Home or Lock screen.
Turn off Portrait Orientation Lock.
Rotate your device to Landscape mode.
Open Settings> Accessibility> Voiceover> Turn on Voiceover.

Steps to reproduce the problem:

1. Open this PDF viewer https://mozilla.github.io/pdf.js/web/viewer.html.
2. Navigate to 'Tools' option at header at header of pdf viewer.
3. Activate 'Open' control and upload any pdf (Having links in it) from your system.
4. Now you'll have the pdf file, in pdf viewer.
5. Navigate through the links and observe the issue.

What is the expected behavior? (add screenshot)
In Landscape mode, voiceover should announce the complete link information such as 'link Securing Information Systems, when user navigates using swipe gestures.

What went wrong? (add screenshot)
Voiceover does not announce the link information while navigating the document in Landscape mode.

In landscape mode voice over announces only 'Link' on each link ('Securing Information Systems link, information security awareness training...) present on PDF using swipe gestures.

https://github.com/gdh1995/pdf.js/assets/93735775/33c783fa-a89f-47ff-a26d-f939b270e5ca

Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension):

**Comment by msftedad** [comment]
Hi Team,Any update on this issue?

**Comment by msftedad** [comment]
Hi @gdh1995, Any update on this issue? Please provide the ETA to fix this issue.
